src/tools/web/README.md

```diff
@@ -0,0 +1,45 @@
+# elektra web
+
+_an API and web user interface to remotely manage multiple Elektra instances_
+
+
+## dependencies
+
+Elektra web requires:
+
+ * Elektra with the `yajl` plugin installed
+ * A recent [node.js](https://nodejs.org/en/) installation (at least 4.x)
+
+
+## Overview
+
+Elektra web consists of multiple components:
+
+ * (multiple) servers running an elektra daemon ([`elektrad`](elektrad/))
+ * a single cluster management server to communicate with the elektra daemons ([`clusterd`](clusterd/))
+ * a client (web browser) that accesses the Web UI on the cluster management server ([`client`](client/))
+
+![https://cdn.rawgit.com/omnidan/libelektra/elektra-web/src/tools/web/doc/network_structure.png](https://cdn.rawgit.com/omnidan/libelektra/elektra-web/src/tools/web/doc/network_structure.png)
+
+
+## GUI
+
+The Web UI allows the user to add new instances to the network, as well as
+combine multiple instances into a cluster. If the configuration of a cluster is
+edited, the changes are pushed to all instances in the cluster. Furthermore,
+single instances can be configured independently.
```

Comment:
Formatted text (e.g. with tool `fmt` or automatic linebreaks from an editor) have the advantage that diffs on them are much nicer.